README.md

```diff
@@ -1473,9 +1469,17 @@ See [Releases](https://github.com/cypress-io/github-action/releases) for full de
 
 *Note: [GitHub announced](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) their plan to disable `save-state` and `set-output` commands by May 31, 2023. This will prevent [cypress-io/github-action](https://github.com/cypress-io/github-action) version [v4.2.1](https://github.com/cypress-io/github-action/releases/tag/v4.2.1), and earlier, running after this date since they use `set-output`. Affected users should update to using `v5` of the [cypress-io/github-action](https://github.com/cypress-io/github-action) action before the deadline.*
 
+## Contributing
+
+[![cypress](https://img.shields.io/endpoint?url=https://cloud.cypress.io/badge/simple/3tb7jn/master&style=flat&logo=cypress)](https://cloud.cypress.io/projects/3tb7jn/runs) [![Action status](https://github.com/cypress-io/github-action/workflows/main/badge.svg?branch=master)](https://github.com/cypress-io/github-action/actions/workflows/main.yml)
```

Comment:
Why add these badges here? I'd expect these to be at the top of the read me?

Reply:
I was trying to mock the Contributing section of Cypress from https://github.com/cypress-io/cypress/blob/develop/README.md#contributing

![image](https://user-images.githubusercontent.com/66998419/230939523-c54a02e8-0f60-4a91-b471-72aaeb833b77.png)

I'd be happy to move them out of this section to the top of the README. In fact the Action status badges is already duplicated at the top, so it would mean just adding the Cypress Cloud badge to the top and removing both badges from Contributing.

Reply:
I moved the badges to the top.

See https://github.com/MikeMcC399/github-action/blob/create/contributing/README.md to review.